addons/website_slides_survey/controllers/main.py

```diff
@@ -1,13 +1,64 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import http
+from odoo import http, _
 from odoo.http import request
+from odoo.exceptions import AccessError
 
+from odoo.addons.website_slides.controllers.main import WebsiteSlides
 
-class WebsiteSlidesSurvey(http.Controller):
-    @http.route(['/slides_survey/certification/search_read'], type='json', auth='user', methods=['POST'], website=True)
-    def slides_certification_search_read(self, fields):
+
+class WebsiteSlidesSurvey(WebsiteSlides):
+    @http.route(['/slides_survey/certification/fetch_certification_info'], type='json', auth='user', methods=['POST'], website=True)
+    def slides_fetch_certification_info(self, fields):
```

Comment:
search_read in a controller?

Maybe this could be replaced with:
- A regular rpc that calls search_read on the JS side
- A check on the user group with something like `this.getSession().user_has_group('survey.survey_user')` instead of this can_create thing

Reply:
Actually relying on real acls is better according to me.

Reply:
What we really need is a `this.getSession().check_access_rights('survey.survey', 'create')`.

But in the mean time we can leave it like that, even though it's a bit cryptic, I agree it's more correct.